frontend/views/components/ConfettiAnimation/AnimationMixins.js

```diff
@@ -282,6 +282,12 @@ const animationMixins = {
   },
   methods: {
     initializeAnimation () {
+      if (!this.$refs.svg) {
+        // This can happen if the user changes page too fast.
+        // It also happens on tests if we don't use a timeout
+        return false
```

Comment:
But [there is a timeout...](https://github.com/okTurtles/group-income-simple/blob/870b86e7f895bbee66fc60b38f837e8c266db5dd/frontend/views/components/ConfettiAnimation/AnimationMixins.js#L352-L355) This shouldn't be here. If there is a problem there must be a better way of doing this.

Reply:
Oh now that you talk about it, that's the true origin of the problem! We are doing a timeout without clearing, for the case when the user leaves the page in less than 500ms. (which happens during the tests) I'll fix that an remove this condition.